js/render.js

```diff
@@ -0,0 +1,18 @@
+import {createPhotosWithDescriptions} from './data.js';
+
+const pictureTemplate = document.querySelector('#picture').content.querySelector('.picture');
```

Comment:
```suggestion
const pictureTemplate = document.querySelector('#picture').content;
```

Так как мы ничего пока что не делаем с элементом `<a href="#" class="picture">`, то самого фрагмента будет достаточно